### ISSUE TITLE
Update ScmDataFrame example notebook

### ISSUE THREAD

**Issue by znicholls** [openclimatedata/openscm#157]
Reminder for me

**Comment by swillner** [comment]
We probably should update and tidy up all notebooks. Maybe have a proper concept aboutwhat notebooks should tell and link from the documentation?
Is this high priority or can we move it to a later milestone?

**Comment by znicholls** [comment]
> Maybe have a proper concept aboutwhat notebooks should tell and link from the documentation?

Yep. I'd suggest they're basically extended examples or they explain concepts which are too technical to go in the docs.

> Is this high priority or can we move it to a later milestone?

I'd move to later milestone (let's use things first, then we can build out the extras)
